data_set/taskset/sanity_check.md

```diff
@@ -0,0 +1,276 @@
+# Tellina sanity check
+Sanity check to ensure that the current version of Tellina can handle this representative set of tasks.
+
+A set of suggestions were tagged as 'helpful` if it contains the answer or a part of the answer. If the suggestions were nonsensical or unusable to solve the task, it was tagged as 'unhelpful'.
+
+- Using original prompts: 8 helpful, 8 unhelpful
+- Adapted prompts: 15 helpful, 1 unhelpful
+
+Tellina has often trouble with vague sentences or ones that have no context, which is why with no modifications, the original prompts do so poorly: They do not represent how programmers would use Tellina. Indeed, with the adapted prompts that are closer to the prompts participants would input in Tellina, we observe much more helpful results.
+
+# Details
+For each sampled post, we try the original prompt as well as an adaptation that's closer on how Tellina will be used during the user experiment.
+
+For each prompt, we report the top 3 suggestions for traceability.
+
+## 1. http://www.bashoneliners.com/oneliners/70/ (Print the lines of file2 that are missing in file1)
+One-liner: `comm -23 file2 file1`
+
+### Prompt: Print the lines of file2 that are missing in file1
+- `comm -1 -3 file1 file2`
+- `sort file1 file2 | uniq`
+- `join < ( sort file2 )`
+
+Helpful
+
+### Adapted prompt: Print the lines in foo.txt that do are not in bar.txt
+- `comm -1 -3 foo.txt bar.txt`
+- `comm -1 -3 foo.txt foo.txt`
+- `join -v 1 foo.txt`
+
+Helpful
+
+## 2. https://www.commandlinefu.com/commands/view/42/display-which-distro-is-installed (Prompt Display which distro is installed) 
```

Comment:
"[...]-is-installed (~Prompt~ Display [...])"

Reply:
Thanks!